pkg/orchestrator/etcd_worker.go

```diff
@@ -362,10 +366,13 @@ func (worker *EtcdWorker) commitChangedState(ctx context.Context, changedState m
 
 	worker.metrics.metricEtcdTxnSize.Observe(float64(size))
 	startTime := time.Now()
-	resp, err := worker.client.Txn(ctx).If(cmps...).Then(ops...).Commit()
+
+	ctx1, cancel := context.WithTimeout(ctx, etcdTxnTimeoutDuration)
+	defer cancel()
+	resp, err := worker.client.Txn(ctx1).If(cmps...).Then(ops...).Commit()
 	costTime := time.Since(startTime).Seconds()
-	if costTime > time.Second.Seconds()*1 {
-		log.Warn("etcdWorker commit etcd txn cost time more than 1 second")
+	if costTime > etcdWorkerLogsWarnDuration.Seconds() {
```

Comment:
What is the motivation for using `Seconds()`?

Reply:
changed to duration.